control/grpc.py

```diff
@@ -485,6 +488,12 @@ def host_count(self):
     def set_ana_group_id(self, anagrpid):
         self.anagrpid = anagrpid
 
+    def set_image_was_shrunk(self, was_shrunk: bool):
+        self.image_was_shrunk = was_shrunk
+
+    def was_image_shrunk(self):
+        return self.image_was_shrunk
+
```

Comment:
this might be cleaner as so:

```suggestion
        @property
    def image_was_shrunk(self):
        return self._image_was_shrunk

    @image_was_shrunk.setter
    def image_was_shrunk(self, value: bool):
        self._image_was_shrunk = value
```

Than we can:
```
ns.image_was_shrunk = True
print(ns.image_was_shrunk)
```

wdyt?

Reply:
I'm not that fluent in the modern python stuff. I'll give it a try.

Reply:
It gave an error and I don't think it's important enough to spend time on that.